base/applications/notepad/lang/sl-SI.rc

```diff
@@ -44,101 +44,101 @@ BEGIN
         MENUITEM "&Prilepi\tCtrl+V", CMD_PASTE
         MENUITEM "Izbri&ši\tDel", CMD_DELETE
         MENUITEM SEPARATOR
-        MENUITEM "&Najdi ...", CMD_SEARCH
+        MENUITEM "P&oišči ...", CMD_SEARCH
         MENUITEM "Na&daljuj iskanje\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Replace...\tCtrl+H", CMD_REPLACE
-        MENUITEM "Go To...\tCtrl+G", CMD_GOTO
+        MENUITEM "Zamenjaj...\tCtrl+H", CMD_REPLACE
+        MENUITEM "Pojdi na...\tCtrl+G", CMD_GOTO
         MENUITEM SEPARATOR
         MENUITEM "Izberi &vse", CMD_SELECT_ALL
-        MENUITEM "Èas/&Datum\tF5", CMD_TIME_DATE
+        MENUITEM "Čas/Dat&um\tF5", CMD_TIME_DATE
```

Comment:
"Datum" should likely not be capitalized. Reversing the order into "Dat&um/čas" would also make it sound more natural, I think.

Reply:
Windows has it that way, but I did Dat&um/čas